week1/index.js

```diff
@@ -1,38 +1,27 @@
-import HTTP from 'http'
-import Path from 'path'
 
-import sendIndexHTML from './responses/sendIndexHTML'
-import sendPage2HTML from './responses/sendPage2HTML'
-import sendStylesCSS from './responses/sendStylesCSS'
-import sendText from './responses/sendText'
+const http = require('http');
+const port = 3000
+var state = 10
 
-const server = HTTP.createServer((request, response) => {
-	console.log(request.method, request.url)
+let server = http.createServer( (request, response) => {
 
-	switch (request.url) {
-	case '/':
-		sendIndexHTML(response)
-		break
-	case '/page2':
-		sendPage2HTML(response)
-		break
-	case '/styles.css':
-		sendStylesCSS(response)
-		break
-	default:
-		const extension = Path.extname(request.url)
-		if (extension === '') {
-			response.statusCode = 302
-			response.setHeader('Location', '/')
-		} else {
-			response.statusCode = 404
-			sendText(response, "File not found")
-		}
-	}
-	
-	response.end()
-})
-
-server.listen(3001)
-
-console.log('Server started')
\ No newline at end of file
+if (request.url === "/state") {
+    response.write(String(state))
+}
+else if (request.url === "/add") {
+    state++;
+    response.write(String(state))
+}
+else if (request.url === "/remove") {
+    state--;
+    response.write(String(state))
+}
+else if (request.url === "/reset") {
+    state = 10;
+    response.write(String(state))
+}
```

Comment:
Very close. But what happens if I request a url other than the ones you have an 'if' condition for?